### ISSUE TITLE
Specify behavior of `<img>` without an image

### ISSUE THREAD

**Issue by tabatkins** [tabatkins/specs#17]
When an `<img>` either has no src, or is currently loading, browsers appear to treat it as 0x0, but this can't be derived from the spec in any way.  Figure out the details and spec it.

**Comment by foolip** [comment]
Yep, also decide which spec gets to define "default object size" for <img>, and the default "default object size" for replaced content for that matter.

**Comment by foolip** [comment]
Oh, also, consider if "default object size" is actually needed. At least for &lt;video>, current Firefox, Chromium and Blink are already (mostly) interoperable in having a default intrinsic size 300x150, which is subtly different from giving it a default object size 300x150, because the former has an intrinsic ratio and the later does not.

**Comment by tabatkins** [comment]
Oh, it's definitely needed.  Perhaps not for `<img>` and maybe not for `<video>`, but images used in CSS definitely have distinct notions of intrinsic and default sizes.

**Comment by foolip** [comment]
OK, so then I suppose it's just a matter of figuring out what values for intrinsic size and default object size are Web compatible for &lt;img> and &lt;video>. Make it so :)

**Comment by tabatkins** [comment]
Closing, because spec is obsolete.

**Comment by foolip** [comment]
Is there a non-obsolete spec which defines this now?

**Comment by tabatkins** [comment]
Oh whoops, I accidentally closed this while closing issues against my src-N spec.  Sorry about that.
